k8s/aat/kured/kured.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: bitnami.com/v1alpha1
+kind: SealedSecret
+metadata:
+  creationTimestamp: null
+  name: kured-values
+  namespace: admin
+spec:
+  encryptedData:
+    values.yaml: AgAoJ4bPZwL7hYvi9V18NJJa/mhqosCCLlLYEuuck1a8E0qE7c5BSzLTFabUo/nzGe7UlR1lJ0GJ9C4rpkIzgwhHa3FUCKTY/LSf6FE6KkE2cFOc+rsvPWTNAbFcTOVpb4AXqgtQABgGvYJzHrfBPJAC3hPdShRmz2YJZnbGZos/kR5UCFCozcEOlnLROkcZdO6hYdIgQ+PDGosLc3MugdE0H5Ks0HDz+6IzxDrv/GaU62tYiEJEnbBhB5DEgHPwW0fFS2z6TjpO7z/MIC4o9t7izn5pzFFMEPqaflaRlSsdRMfzO6fjodzd0fmNsKseL7sXrvvfDDOxpBF0rjSAhrNdSSi2HQ+cwjuSAKG56Bfox7rtn9mvAmSy8Z4nSeOo7jnawviozbtunr6klf+jaPvHbHVabdyS1TRaZ1thTgrdmLzFS/zafZW7fcfWCR4baa1lSF2o7vb5UFB0h2lUWWqVF1PspbvbJ30I+QXpj26vauRQ4DMMNDuY8Htb4eRY/Mw/D0uXx9nneMBc/T9a+i5l8bFcc2fv6ToA/VeQ+W37J82eOrNVKWJFQFtJcRHETt9J3Z73a0NihB4QDtWNJfAj+r1kRsxifpyh5pBblf2tA8vHayU8nhlKhZucJr2kEPi6m617OyVRH5l367eLdQJyl4PAmW5jcUoc50OWDfEnxxZtnCNPwt3+7P4WgGTZQZPY3N09XuaFcHLxVNzFblrlgw8raE1gEAMpFKeWOkkwzHIssnNGJRoZqiz9zbXZPr69pMPW8mCAEtXlejc8eb/PdiAnJBQeqxX5IPOEsFI6SXLsSpxBJOWnorXQHnpSIuGFY4HAqQzkp+8EBgLce/yQmztaU3s1qB5XrOw2sg==
+
```

Comment:
do we want to name these to indicate their type?  e.g. `kured-creds.yaml` or something?